Add Ṣifhu Lī

adehad opened the pull request.

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ جَمِيلَ القَدِّ وَرْدِيَّ الوِجَانَا "],
+            ["صِفْ مَلِيحًا حُسْنُهُ فَاقَ الخَيَالًا"],
+
+            ["صِفْ بِهِ عُنُقًا مُنِيرًا كَوْكَبِِيًّا"],
```
["صِف بِهِ عُنُقاً مُنِيراً كَوكَبِياً"] ,

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ مَلِيحًا حُسْنُهُ فَاقَ الخَيَالًا"],
+
+            ["صِفْ بِهِ عُنُقًا مُنِيرًا كَوْكَبِِيًّا"],
+            ["صِفْ جَمِيْلًا نُورُهُ فِي الكَوْنِِ لَالَا"],
```
["صِفْ جَمِيْلًا نُورُهُ فِي الكَونِ لَالَا"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ ضَلِيعَ الفَمِ بَرَّاقَ الثَّنَايَا"],
+            ["صِفْ نَدِيَّ الرَّاحِ مِنْهُ الغَيْتُ سَالَا"],
+
+            ["صِفْ طَوِيلَ الهُدْبِ صِفْ أَنْفًا كَسَيْفٍ "],
```
["صِفْ طَوِيلَ الهُدْبِ صِفْ أنْفَاً كَسَيْفٍ "],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ نَدِيَّ الرَّاحِ مِنْهُ الغَيْتُ سَالَا"],
+
+            ["صِفْ طَوِيلَ الهُدْبِ صِفْ أَنْفًا كَسَيْفٍ "],
+            ["صِفْ أَزَجَّ الحَاجِبَيْنِِ بِلَا اتِّصالَا"],
```
["صِفْ أَزَجَّ الحَاجِبَيْنِ بِلَا اتِّصالَاً"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ طَوِيلَ الهُدْبِ صِفْ أَنْفًا كَسَيْفٍ "],
+            ["صِفْ أَزَجَّ الحَاجِبَيْنِِ بِلَا اتِّصالَا"],
+
+            ["صِفْ نَبِيًّا قَدْ أَتَى مِنْ قَبْلِ آدمْ "],
```
["صِفْ نَبِيّاً قَدْ أَتَى مِنْ قَبْلِ آدَمْ "],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ أَزَجَّ الحَاجِبَيْنِِ بِلَا اتِّصالَا"],
+
+            ["صِفْ نَبِيًّا قَدْ أَتَى مِنْ قَبْلِ آدمْ "],
+            ["جَلَّ مَنْ سَوَّاهُ لَيْسَ لَهُ مِثَالَا"],
```
["جَلَّ مَنْ سَوَّاهُ لَيْسَ لَهُ مِثَالَاً"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["قُلْ بِرَبِّكَ كَيْفَ أَدْرَكْتَ الوِصَالَا"],
+
+            ["كَمْ أُنَادِي يَا أَبَا الزَهْرَاءْ أَقْبِل"],
+            ["كَمْ أٌنادٍي يَا أَبَا الزَهْرَا تَعَالَ"],
```
["كَمْ أُنَادِي يَا أَبَا الزَهرَاء تَعَالَ"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["كَمْ أُنَادِي يَا أَبَا الزَهْرَاءْ أَقْبِل"],
+            ["كَمْ أٌنادٍي يَا أَبَا الزَهْرَا تَعَالَ"],
+
+            ["صِحْتُ وَاشَوْقِاه وَجْداً يَا حَبِيبِي"],
```
["صِحْتُ وَاشَوْقَاه وَجْداً يَا حَبِيبِي"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+    },
+    "verses": {
+        "ara": [
+            ["صِفْهُ لِي يَا مَنْ رَأَيْتَ الحِبَّ لَيْلًا"],
```
["صِفْهُ لِي يَا مَنْ رَأَيْتَ الحِبَّ لَيْلاً"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+    "verses": {
+        "ara": [
+            ["صِفْهُ لِي يَا مَنْ رَأَيْتَ الحِبَّ لَيْلًا"],
+            ["إِنَّ عَيْنِي تَشْتَهِي ذَاكَ الجَمَالَا"],
```
["إِنَّ عَيْنِي تَشْتَهِي ذَاكَ الجَمَالَاً"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْهُ لِي يَا مَنْ رَأَيْتَ الحِبَّ لَيْلًا"],
+            ["إِنَّ عَيْنِي تَشْتَهِي ذَاكَ الجَمَالَا"],
+
+            ["صِفْ جَمِيلًا كَامِلًا خَلْقًا وَخُلُقًا"],
```
["صِفْ جَمِيلاً كَامِلاً خَلْقاً وَخُلُقاً"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["صِفْ جَمِيلًا كَامِلًا خَلْقًا وَخُلُقًا"],
+            ["مِنْ جَمَالِ الذَّاتِ قَدْ حَازَ الكَمَالَا"],
+
+            ["صِفْ مَلِيحًا طَرْفُهُ أَسْيَى العَوَالِمْ "],
```
["صِفْ مَلِيحاً طَرْفُهُ أسْبَى العَوَالِمْ "],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["مِنْ جَمَالِ الذَّاتِ قَدْ حَازَ الكَمَالَا"],
+
+            ["صِفْ مَلِيحًا طَرْفُهُ أَسْيَى العَوَالِمْ "],
+            ["صِفْ وَجِيهًا وَجْهُهُ حَازَ الكَمَالَا"],
```
["صِفْ وَجِيهاً وَجْهُهُ حَازَ الكَمَالَا"],

thenotepadwriter commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["كَامِلَ الأَوْصَافِ قَدْ مَلَكَ الدَّلَالَا"],
+
+            ["صِفْ عُيُونَ الهَاشِمِي صِفْ لِي المُحَيَّا"],
+            ["صِفْ لِي ثَغْرًا بِابْتِسَامَتِهِ تَلَالًا"],
```
["صِفْ لِي ثَغْراً بِابْتِسَامَتِهِ تَلَالا"],

adehad replied:
did this one mean to drop the fatha on the end?
ie would it be
```suggestion
            ["صِفْ لِي ثَغْراً بِابْتِسَامَتِهِ تَلَالَا"],
```

thenotepadwriter reviewed (changes requested):
Salaams!

I only looked at the Arabic, and made edits to the harakat. Just note that it looks like Sayed Maan uses an automatic tashkeel tool (w Allahu a'lam), and it's not very accurate. 2 tiny tips come to my mind which might be helfpul in future:

- The tanween ً  only ever comes on the last letter of a word (if it's anywhere else, you know it's wrong!) 
- The letter ا (alef) can only ever be preceded with a fath َ so if the letter before it has any other harakah, you can also immediately tell it's not right...

adehad commented on site/data/qasida/sifhu-li-ya-man-ra-ayta.json:
```diff
@@ -0,0 +1,214 @@
+            ["أَنْجَلًا تُنْسِيكَ طَلْعَتُهُ الهِلَالَا"],
+
+            ["صِفْ جَمِيلَ القَدِّ وَرْدِيَّ الوِجَانَا "],
+            ["صِفْ مَلِيحًا حُسْنُهُ فَاقَ الخَيَالًا"],
```
also this one?

```suggestion
            ["صِفْ مَلِيحاً حُسْنُهُ فَاقَ الخَيَالَا"],
```

adehad reviewed (commented):
Wa Salaams wa BarakAllah feekunna!

Sorry I edited your comments so they showed the before and after as I was struggling to see the differences sometimes haha
![image](https://github.com/user-attachments/assets/cc80694a-6bd1-4957-b38e-c898dbc14ed9)

I think there are a few I have some questions about the ending on this, but I can imagine that `لا` is only read in one way so maybe that's why it is less of an issue?

thenotepadwriter commented:
Wa alaykum assalam, wa fekum barak!

Ah, can you show me how to do those before/after comments in future (from the Android app...)?

I will look at the comments now isA

thenotepadwriter commented:
I'm sorry I'm struggling to reply to the comments directly (see screenshot), not sure why. But here are my replies:

![Screenshot_20240819_180106_GitHub.jpg](https://github.com/user-attachments/assets/d1d4c539-be3c-47ee-a9d3-254d54fcac50)

There are 2 changes here:

ثغراً - The tanween moves from the penultimate letter to the last letter

تلالَا - The tanween is removed from the penultimate letter and the fath remains

There are 2 changes here:

1. مليحاً - The tanween moves to the last letter

2. الخيالَا - The tanween is removed and we keep the fath

thenotepadwriter commented:
And if this explanation helps at all:

There are 2 different لا endings used in this Qasida, some need a tanween (which is not pronounced when we stop on the word, and would only be pronounced if we joined it with the next word). The other kind does not need a tanween, that's why I added it in some places and removed it in others

adehad commented:
> And if this explanation helps at all:
> 
> There are 2 different لا endings used in this Qasida, some need a tanween (which is not pronounced when we stop on the word, and would only be pronounced if we joined it with the next word). The other kind does not need a tanween, that's why I added it in some places and removed it in others

this does make a lot of sense thanks

> I'm sorry I'm struggling to reply to the comments directly (see screenshot), not sure why. But here are my replies:
> 
> ![Screenshot_20240819_180106_GitHub.jpg](https://private-user-images.githubusercontent.com/70443170/359185339-d1d4c539-be3c-47ee-a9d3-254d54fcac50.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwODgzOTUsIm5iZiI6MTcyNDA4ODA5NSwicGF0aCI6Ii83MDQ0MzE3MC8zNTkxODUzMzktZDFkNGM1MzktYmUzYy00N2VlLWE5ZDMtMjU0ZDU0ZmNhYzUwLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDE3MjEzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmNDQxMDFmZmVkZDE2ZDMxZTU2OWE2OTFjNmZhMjZjNzQzMjJjNThmMTY0ZTBhZTAzMmUwMjhlMjNkNDU2NTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ot5RST16q__WFmTMRakemj-9oIhc48b-AWk9z38f388)

this is a mystery to me as well


> 
> There are 2 changes here:
> 
> ثغراً - The tanween moves from the penultimate letter to the last letter
> 
> تلالَا - The tanween is removed from the penultimate letter and the fath remains
> 
> There are 2 changes here:
> 
>     1. مليحاً - The tanween moves to the last letter
> 
>     2. الخيالَا - The tanween is removed and we keep the fath

perfect



> Ah, can you show me how to do those before/after comments in future (from the Android app...)?
> 

I think it is also the same `±` symbol in the mobile app to look out for

thenotepadwriter commented:
Khair, jazakum Allah khairan! & bismillah..
